samples/shell/echo/echo.sh

```diff
@@ -1 +1,3 @@
-echo "received: $1"
+#!/bin/sh
+
+echo -n `cat`
```

Comment:
This doesn't add "received:".

Reply:
Maybe add ``date`` or something similar here, so that each invocation can be obviously different

Reply:
Good idea. Will do.